parl/core/torch/tests/model_base_test_torch.py

```diff
@@ -331,7 +337,7 @@ def test_set_weights_wrong_params_num(self):
 
     def test_set_weights_wrong_params_shape(self):
         params = self.model.get_weights()
-        params.reverse()
+        params['fc1.weight'] = params['fc2.bias']
         try:
```

Comment:
Please use the [TestCase.assertRaises](https://stackoverflow.com/questions/129507/how-do-you-test-that-a-python-function-throws-an-exception) API to test the exception.